src/NLU.DevOps.LuisV3/LuisNLUTestClient.cs

```diff
@@ -40,15 +46,23 @@ public LuisNLUTestClient(LuisSettings luisSettings, ILuisTestClient luisClient)
             JToken query,
             CancellationToken cancellationToken)
         {
-            if (query == null)
+            try
             {
-                throw new ArgumentNullException(nameof(query));
-            }
+                if (query == null)
+                {
+                    throw new ArgumentNullException(nameof(query));
+                }
 
-            var predictionRequest = query.ToObject<PredictionRequest>();
-            predictionRequest.Query = predictionRequest.Query ?? query.Value<string>("text");
-            var luisResult = await this.LuisClient.QueryAsync(predictionRequest, cancellationToken).ConfigureAwait(false);
-            return this.LuisResultToLabeledUtterance(new SpeechPredictionResponse(luisResult, 0));
+                var predictionRequest = query.ToObject<PredictionRequest>();
+                predictionRequest.Query = predictionRequest.Query ?? query.Value<string>("text");
+                var luisResult = await this.LuisClient.QueryAsync(predictionRequest, cancellationToken).ConfigureAwait(false);
+                return this.LuisResultToLabeledUtterance(new SpeechPredictionResponse(luisResult, 0));
+            }
+            catch (ErrorException ex)
+            {
+                Logger.LogError(ex.Body.ErrorProperty.Message);
```

Comment:
Can you also log the HTTP status code and any other pertinent error details, e.g.:
```
Logger.LogError($"Received error with status code '{ex.Body.StatusCode}' and message '{ex.Body.Message}'.");
```

Reply:
done.